lib/legacy/polymer.dom.html

```diff
@@ -235,6 +235,25 @@
     }
   }
 
+  /** @type {function(boolean=):Node} */
+  DomApi.prototype.cloneNode;
+  /** @type {function(Node):!Node} */
+  DomApi.prototype.appendChild;
+  /** @type {function(Node, Node):!Node} */
+  DomApi.prototype.insertBefore;
+  /** @type {function(Node):!Node} */
+  DomApi.prototype.removeChild;
+  /** @type {function(Node, Node):!Node} */
+  DomApi.prototype.replaceChild;
+  /** @type {function(string, (string|number|boolean)):void} */
+  DomApi.prototype.setAttribute;
+  /** @type {function(string):void} */
+  DomApi.prototype.removeAttribute;
+  /** @type {function(string):?Element} */
+  DomApi.prototype.querySelector;
+  /** @type {function(string):!NodeList<!Element>} */
+  DomApi.prototype.querySelectorAll;
```

Comment:
how do we want to handle these? this seems the right way to define them but the generator doesn't seem to pick them up.

Reply:
I don't think we currently have a way to tell Analyzer about methods added to a class after declaration. @rictic is there any trick here?